redis/commands/core.py

```diff
@@ -4811,6 +4811,126 @@ def readonly(self, **kwargs):
         return self.execute_command("READONLY", **kwargs)
 
 
+class FunctionCommands:
+    """
+    Redis Function commands
+    """
+
+    def function_load(
+        self,
+        engine: str,
+        library: str,
+        code: str,
+        replace: Optional[bool] = False,
+        description: Optional[str] = None,
+    ) -> str:
```

Comment:
Since we return OK from Redis, isn't this a bool? See the unit test asserts as an example - given that we handle the parsing.